src/components/loading/loading.hbs

```diff
@@ -2,8 +2,10 @@
 <div data-loading class="{{@root.prefix}}--loading{{#if small}} {{@root.prefix}}--loading--small{{/if}}">
   <svg class="{{@root.prefix}}--loading__svg" viewBox="-75 -75 150 150">
     <title>Loading</title>
+    {{#if componentsX}}
     {{#if small}}<circle class="{{@root.prefix}}--loading__background" cx="0" cy="0" r="37.5" />{{/if}}
-    <circle class="{{@root.prefix}}--loading__stroke" cx="0" cy="0" r="37.5" />
+    {{/if}}
+    <circle {{#if componentsX}}class="{{@root.prefix}}--loading__stroke"{{/if}} cx="0" cy="0" r="37.5" />
```

Comment:
Wanted to double-check if we are making `bx--loading__stroke` experimental only.

Reply:
this was a class I added in during the experimental changes of the loading therefore would be a breaking change added a new class. So for now yes should be experimental only same with background as they are new classes not present in older markup

Reply:
It was my mistake in first round forgetting to add the if statement